docs/source/support/iroha.md

```diff
@@ -0,0 +1,17 @@
+Iroha Support
+-----------------
+
+```{note}
+The deployContract feature not yet implemented since Iroha lacks full smart contract support during the initial development stage of the Iroha connector plugin.
+```
+
+<details>
+  <summary>Hyperledger Cactus v0.8.0</summary>
+
+  | Iroha version | runTransaction |
+  | --- | :---: |
+  | Iroha 1.2.0 and Postgres 9.5 | ✅ [test](https://github.com/hyperledger/cactus/blob/v0.6.0/packages/cactus-plugin-ledger-connector-iroha/src/test/typescript/integration/run-transaction-endpoint-v1.test.ts) |
```

Comment:
@hxlaf Link is dead to the test, but I understand right now it's not possible to pre-determine what the valid link will be so I recommend opening a follow-up issue to track the task of updating this link in the docs once the current PR gets merged/released.

Reply:
Need to talk to you tomorrow about it!

Reply:
[(Github issue)](https://github.com/hyperledger/cactus/issues/1246) has been posted!
Thank you!